native/packages/react-native-bpk-component-icon/readme.md

```diff
@@ -8,9 +8,20 @@
 npm install react-native-bpk-component-icon --save-dev
 ```
 
-<!-- // TODO ADD INSTRUCTIONS FOR ADDING THE PODSPEC / GRADLE SCRIPT TO THE iOS / Android PROJECT IN WHICH THIS IS NOW BEING USED! -->
+## Native usage
 
-## Usage
+### IOS
+
+Add the following to your `Podfile` and run `pod update`:
+
+`pod 'BpkIcons', :path => '../react-native/node_modules/react-native-bpk-component-icon'`
```

Comment:
Why:
```
'../react-native/node_modules/react-native-bpk-component-icon'
```
and not:
```
'node_modules/react-native-bpk-component-icon'
```

I.e. surely it depends on the consumer's project to determine the correct path?

(i'm referencing https://github.com/oblador/react-native-vector-icons#option-with-cocoapods)

Reply:
I followed the same logic as the other RN components used in the app, both IOS and Android have a react-native folder in the root so that's why the `../`

Reply:
Yeah but that's specific to our app, we shouldn't be encoding that here IMO

Reply:
ok no problem, I'll amend that